io/src/main/scala/fs2/io/ToInputStream.scala

```diff
@@ -0,0 +1,259 @@
+package fs2.io
+
+
+import java.io.{IOException, InputStream}
+
+
+import fs2._
+import Chunk.Bytes
+import util.{Async, Attempt}
+import async.mutable
+import Async.Change
+
+import scala.concurrent.SyncVar
+
+
+/**
+  * Conversion of Stream[F,Byte] to java.io.InputStream
+  */
+object ToInputStream {
```

Comment:
Was my original thought too. I am not sure if it is good idea then as there is no reuse/relation to other methods there, and I doubt ever will be so, I was kinda of preventing the clutter in unrealted code. But is ok for me to do that if you think it makes sense